corva/event/event.py

```diff
@@ -0,0 +1,21 @@
+from __future__ import annotations
+
+from collections import UserList
+from typing import List
+
+from corva.event.data.base import BaseEventData
+
+
+class Event(UserList):
+    def __init__(self, data: List[BaseEventData]):
+        super().__init__(data)
+
+    def __eq__(self, other):
```

Comment:
why do we need to implement `__eq__` method?

Reply:
Actually, it is not needed, will delete it